CRYSTAL23 Installation.

rkalescky opened the pull request.

Commit:
Added Crystal module.

Commit:
Added CRYSTAL23.

jrlagrone commented on docs/examples/crystal/be_sto3g/crystal.sbatch:
```diff
@@ -1,22 +1,17 @@
 
 module purge
-module load crystal/17
+module use /users/rkalescky/hpc_docs/modules/m3/applications
```
Do you want your path in the example?

Commit:
Update crystal.sbatch

- Removed testing module path.
- Added description of OpenMP and OpenMP+MPI variants.
